spanner/spanner_snippets/graph_snippet.go

```diff
@@ -0,0 +1,486 @@
+// Copyright 2024 Google LLC
```

Comment:
issue: break these samples into separate files, please.

See:
https://googlecloudplatform.github.io/samples-style-guide/#one-per-file

Reply:
Done. Thanks for the pointer to this guide.